platform/datacollector/src/main.rs

```diff
@@ -40,7 +45,24 @@ async fn data_handler() -> impl Responder {
 
     data_client.write_equities_bars(new_bars).await.unwrap();
 
-    HttpResponse::Ok().body("OK")
+    let response_event= EventBuilderV10::new()
+        .id(Uuid::new_v4().to_string())
+        .ty("data.ohlcv.updated")
+        .source("psf.platform.datacollector")
+        .data(
+            "application/cloudevents+json",
+            json!({
+                "status": "success".to_string(),
```

Comment:
not strictly necessary, but as we start porting events into the library, I would think about having a status enum like

```rust
use std::fmt;
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub enum Status {
    Failure,
    Success,
    Unknown
}

impl fmt::Display for Status {
    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
        match self {
            Success => write!("success"),
            Failure => write!("failure"),
            unknown => write!("unknown")
        }
    }
}
```

I have something similar in the market hours check: https://github.com/pocketsizefund/pocketsizefund/blob/257e3156f478dd269b7e97ed6da9c2eba56de49f/libraries/rust/src/events/market.rs#L10